crates/cranelift/src/lib.rs

```diff
@@ -33,8 +33,24 @@ mod func_environ;
 mod gc;
 mod translate;
 
-/// User trap code for debug assertions we emit in our JIT code.
-const DEBUG_ASSERT_TRAP_CODE: u16 = u16::MAX;
+const TRAP_DEBUG_ASSERT: TrapCode = TrapCode::user(1);
```

Comment:
What do you think about renaming this to `TRAP_INTERNAL_ASSERT` while we're here? We emit this not just in debug builds but also in release builds for certain asserts, so the "debug" nomenclature has been kind of nagging at me recently.